web/src/engine/websites/RawLazy.ts

```diff
@@ -0,0 +1,101 @@
+﻿import { Tags } from '../Tags';
+import icon from './RawLazy.webp';
+import { type Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin, type MangaScraper } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchJSON, FetchWindowScript } from '../platform/FetchProvider';
+
+function MangaLabelExtractor(element: HTMLElement) {
+    const text = element instanceof HTMLAnchorElement ? element.text : element.textContent.split('|')[0].trim();
+    return text.replace(/\(Raw.*Free\)/i, '').trim();
+}
+
+function ChapterExtractor(this: MangaScraper, anchor: HTMLAnchorElement) {
+    return {
+        id: anchor.pathname,
+        title: anchor.querySelector('span').textContent.trim()
+    };
+}
+
+const findNextStageScript = `
+    new Promise(resolve =>  {
+        const interval = setInterval(function () {
+            if(jQuery('{BUTTONID}').attr('href') != '#'){
+                clearInterval(interval);
+                resolve(jQuery('{BUTTONID}').attr('href'));
```

Comment:
May use `querySelector` and expanded `href` property instead of JQuery and non-expanded `href` attribute

Reply:
Yeah. There is no pb with href tho, since this always another host =  a fully fledged url.